usaspending_api/etl/tests/test_load_submission.py

```diff
@@ -64,3 +64,18 @@ def test_load_submission_command(endpoint_data, partially_flushed):
     assert sub.reporting_fiscal_quarter == 1
     assert sub.reporting_period_start == date(2015, 10, 1)
     assert sub.reporting_period_end == date(2015, 12, 31)
+
+
+@pytest.mark.django_db
+def test_load_submission_command_program_activity_uniqueness(endpoint_data, partially_flushed):
```

Comment:
Thanks for adding this. Good to see us adding some more unit tests on the submission load process, given that we're about to kick it into high gear!